vNext/src/Orleans/OrleansLogging/FileLogConsumer.cs

```diff
@@ -0,0 +1,85 @@
+﻿using Orleans.Runtime;
+using System;
+using System.Collections.Generic;
+using System.Diagnostics;
+using System.IO;
+using System.Net;
+using System.Text;
+using System.Threading;
+
+namespace Orleans.Extensions.Logging
+{
+    /// <summary>
+    /// FileLogConsumer, which logs message into a file in orleans logging message style
+    /// </summary>
+    public class FileLogConsumer : ICloseableLogConsumer, IFlushableLogConsumer
```

Comment:
`FileLogConsumer` is the replacement for `FileTelemetryConsumer`. FileTelemetryConsumer is the default component in Orleans to record tracing messages. But since we are separating logging and telemetry, so FileTelemetryConsumer won't be compatible with asp.net logging abstraction anymore. I created FileLogConsumer to replace its role in logging, just in case that people still want to log messages in Orleans logging message style.